src/main/java/org/zalando/fahrschein/Listener.java

```diff
@@ -4,6 +4,9 @@
 import java.util.List;
 
 public interface Listener<T> {
-    void onEvent(List<T> event) throws IOException, EventAlreadyProcessedException;
+    void onEvent(final List<T> event) throws IOException, EventAlreadyProcessedException;
 
+    default void onError(final IOException e) throws IOException {
+        // no operation
```

Comment:
Default should probably be to throw the exceptions instead of silently swallowing it.